src/.vuepress/algolia_index.js

```diff
@@ -0,0 +1,140 @@
+const algoliasearch = require('algoliasearch')
+const glob = require("glob")
+const {
+  readFileSync
+} = require("fs")
+const client = algoliasearch('2MY54GAA5W', '5ba37f0a8d8da6bbbe202e6707ca1998')
```

Comment:
@zatkniz - what if we put the creds in .env file instead of hardcode it in js file?

Reply:
I think I said that last time and but we concluded don't waste any more time on that! I wanted to add it in a TODO as well but forgot to add it. I believe it's a very bad choice to add credentials to a public project and I think we should delete the PR and remove history. Credentials were exposed as well in the previous version. Please let me know so I can proceed with .env implementation

Reply:
Yes, move it to env var and sync with @SimoneLazzaris because this will require update in deployment before merge

Reply:
Replaced with .env